calculator.js

```diff
@@ -23,6 +28,63 @@ const calculateUserInput = function (error, promptInput) {
 //   operation: 'add',
 // });
 
+const calculateUserInput = function(error, promptInput){
+  const operation = promptInput.operation
+  const firstNum = parseInt(promptInput.num1)
+  const secondNum = parseInt(promptInput.num2)
+  const operators = ['add', '+', 'subtract', '-', 'divide', '/', 'multiply', '*']
+  
+  if (isNaN(firstNum)) {
+    console.log(`${promptInput.num1} is not a number.` )
+    return
+  } else if (isNaN(secondNum)) {
+    console.log(`${promptInput.num2} is not a number.` )
+    return
+  } else if (!operators.includes(operation)) {
+    console.log('unsupported operator')
+    return 
+  } else if (operation === 'add' || operation === '+') {
+    addNums(firstNum, secondNum)
+  } else if (operation === 'subtract' || operation === '-') {
+    subtractNums(firstNum, secondNum)
+  } else if (operation === 'multiply' || operation === '*') {
+    multiplyNums(firstNum, secondNum)
+  } else if (operation === 'divide' || operation === '/') {
+    divideNums(firstNum, secondNum) 
+  } else {
+    console.log('Exiting the Program');
```

Comment:
This message doesn't make it clear that the program is exiting because of a bad operator.